### PR TITLE
Add an improved version of the gif in README

### PR DESCRIPTION
**Pull request by AndyTitu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ No more plaintext credentials in your home directory. Automatically authenticate
 <p align="center">
   <picture>
   <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/45081667/227191964-9629476d-a49e-475d-b8cb-2115c302025d.gif">
-   <img src="https://user-images.githubusercontent.com/45081667/229802932-d3a5de19-82f1-43bf-825d-befcc649ed29.gif" alt="Example of 1Password Shell Plugins with AWS: user runs an `aws` command, a Touch ID prompt shows up, and `aws` is automatically authenticated" style="max-width: 100%; display: inline-block;" />
+   <img src="https://user-images.githubusercontent.com/45081667/232083691-2bee9a7e-4ec5-4e30-966f-3511cbbd2142.gif" alt="Example of 1Password Shell Plugins with AWS: user runs an `aws` command, a Touch ID prompt shows up, and `aws` is automatically authenticated" style="max-width: 100%; display: inline-block;" />
```
Comment (florisvdg) [inline README.md:29]: This image adds a gray line at the bottom:

image

**Review by florisvdg — COMMENTED** [review]
Way better, thanks! 😍 

Left one comment below.

**Comment by AndyTitu** [comment]
In the process of uploading the web format (without a new line an the bottom) to `developer.1password.com`.

File: README.md (hunk 1 above)
Comment (AndyTitu) [reply README.md]: The new addition removes that line

**Comment by AndyTitu** [comment]
Added the new version!

**Review by florisvdg — APPROVED** [review]
Beautiful! 😍

**Review by jpcoenen — APPROVED** [review]

**Pull request merged**